app/services/settings-v2/video.ts

```diff
@@ -0,0 +1,206 @@
+import * as remote from '@electron/remote';
+import { Inject } from 'services/core/injector';
+import { InitAfter } from 'services/core';
+import { mutation, StatefulService } from '../core/stateful-service';
+import * as obs from '../../../obs-api';
+import { SettingsManagerService } from 'services/settings-manager';
+import { $t } from 'services/i18n';
+
+// export interface IVideo {
+//   fpsNum: number;
+//   fpsDen: number;
+//   baseWidth: number;
+//   baseHeight: number;
+//   outputWidth: number;
+//   outputHeight: number;
+//   outputFormat: EVideoFormat;
+//   colorspace: EColorSpace;
+//   range: ERangeType;
+//   scaleType: EScaleType;
+//   fpsType: EFPSType;
+// }
```

Comment:
I take it this is commented out since you're using the interface exposed by OSN instead?  If so, should we delete this?

Reply:
This was copied over from the OSN file just for ease of my reference :) will remove